Azure/sonic-buildimage
# Add QoS and Buffers configs for Mellanox MSN2100 and MSN2410

**- What I did**
Added QoS and Buffers configs for Mellanox MSN2100 and MSN2410

**- How I did it**
Added files to corresponding folders

**- How to verify it**
Install on MSN2100 or MSN2410 and make sure buffers configuration is generated and applied
